l10n_es_pos_oca/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+    <t
+        t-name="OrderReceipt"
+        t-inherit="point_of_sale.OrderReceipt"
+        t-inherit-mode="extension"
+    >
+        <xpath expr="//t[@t-if='receipt.company.contact_address']" position="before">
+            <t t-if="env.pos.config.is_simplified_config and !receipt.to_invoice">
+                <div><t t-esc='receipt.date.localestring' /></div>
```

Comment:
t-esc directives are also deprecated, but they still works as an alias of t-out. When possible, switch to this new name in advance of the future directive removal.

Reply:
En plantillas estáticas hay matices: https://github.com/odoo/odoo/pull/168191#issuecomment-2199431042